bokehjs/src/coffee/models/renderers/glyph_renderer.coffee

```diff
@@ -16,22 +16,22 @@ class GlyphRendererView extends Renderer.View
 
     selection_glyph = @mget("selection_glyph")
     if not selection_glyph?
-      attrs = _.extend {}, glyph_attrs, @model.selection_defaults
-      selection_glyph = new (@mget("glyph")).constructor(attrs)
+      #attrs = _.extend {}, glyph_attrs, @model.selection_defaults
```

Comment:
I guess there is an implicit assumption (not enforced) that they are all the *same* glyph type (up to an equivalence class of "marker", those should be freely interchangeable)